beep/featurize.py

```diff
@@ -137,7 +137,12 @@ def init_full_model(cls, processed_cycler_run, init_pred_cycle=10, mid_pred_cycl
         assert len(processed_cycler_run.summary) > final_pred_cycle, 'cycle count must exceed final_pred_cycle'
         cycles_to_average_over = 40  # For nominal capacity, use median discharge capacity of first n cycles
 
-        interpolated_df = processed_cycler_run.cycles_interpolated
+        if 'step_type' in processed_cycler_run.cycles_interpolated.columns:
```

Comment:
Curious to know when this "if" is triggered. Are there occasional cycles_interpolated that are unlabeled, but you know that they're discharge by default? Or are the cycles something entirely different?

Reply:
Yeah, good point. I will add a comment. 
The functionality here is the same as it was, it's just that since we are adding charge steps to the interpolated data we need to filter the data for the discharge steps to get the same outcome as before. Benben should be adding some features that are dependent on the charge step in the regular cycles, so we will probably include a different feature set at that point.